keras/utils/generic_utils.py

```diff
@@ -183,6 +183,25 @@ def func_dump(func):
     return code, defaults, closure
 
 
+def ensure_value_to_cell(value):
```

Comment:
This should be private. Also, since this function is only called once (in a loop), please consider if you could in-line it in the parent function.

Reply:
Just to clarify:
1) By making it private, do you mean 
   a) adding a "_" before the name, or 
   b) moving the function inside the function to which it is considered private?

2) I could inline the function, but I feel it might impact the readability of `def func_load` with negligible savings (the closure list should always be 1 or 2 items long in practice).
 I don't mind either way; which alternative would you prefer?

Reply:
What if you replace the list comprehension with an explicit loop? That sounds like it could be more readable.

Reply:
I've made a gist where it's as an explicit loop: https://gist.github.com/rh314/0fdb8341123ea0fea8443f5cde496858
(Covers both the non-inlined (option 1) and the inlined (option 2) cases)

Thoughts on which one to use?

Reply:
So I think we should:
-  move the function inside the parent function, since no one else will use it
- keep the list comprehension, since we won't in-line the function after all...

Thanks!

Reply:
Done!
PS: Could you please re-run Travis, it failed/flaked with "Exception: URL fetch failure on https://www.cs.toronto.edu/~kriz/cifar-100-python.tar.gz"